### ISSUE TITLE
Check UTF8 sanitization

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/omnicore#438]
Right now, while the [functions](https://github.com/OmniLayer/omnicore/blob/master/src/omnicore/mbstring.cpp) for proper UTF8 sanitization are available, RPCs are currently not explicitly sanitized, because I first wanted to check the behavior of `UniValue`.

We should check whether we still need the sanitization (e.g. by following the steps described in #108).

**Comment by dexX7** [comment]
Looks like we need to the sanitization after all:

```
$ ./src/omnicore-cli omni_getproperty 2147483662
error: couldn't parse reply from server
```

**Comment by dexX7** [comment]
Resolved via #443.
